test/dfusionMultiSafes.js

```diff
@@ -544,6 +554,59 @@ contract("GnosisSafe", function(accounts) {
         assert(sellOrder.priceNumerator.eq(max128))
       }
     })
+    it("Places bracket orders on behalf of a fleet of safes and checks prices for p<1, with different decimals than 18", async () => {
+      const masterSafe = await GnosisSafe.at(
+        await deploySafe(gnosisSafeMasterCopy, proxyFactory, [lw.accounts[0], lw.accounts[1]], 2)
+      )
+      const bracketSafes = await deployFleetOfSafes(masterSafe.address, 6)
+      testToken = await TestToken.new("TEST6", 6)
+      const testToken2 = await TestToken.new("TEST4", 4)
+      const targetToken = 0 // "TEST4"
```

Comment:
`targetToken` is the target token id, right? But then if the id is zero we are building the orders against OWL instead of the new token. To add a new token to the exchange and get back the token and the id you could use the function `addCustomMintableTokenToExchange` from PR #107.

Reply:
very good catch! Thanks for reviewing it so carefully.